lib/src/booleans.dart

```diff
@@ -0,0 +1,9 @@
+/// True if the boolean [value] is false
```

Comment:
Use graves around code terms
```suggestion
/// `true` if the boolean [value] is `false`
```